src/screens/WebsocketResearchScreen/SignedChatScreen.tsx

```diff
@@ -86,7 +83,7 @@ const SignedChatScreen = () => {
       <View style={styles.inputContainer}>
         <AnonymousInputMessage onSendButtonClicked={sendChat} type={SIGNED} />
       </View>
-    </KeyboardAvoidingView>
+    </View>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The closing tag for `KeyboardAvoidingView` has been replaced with the closing tag for `View`. This change is consistent with the replacement of `KeyboardAvoidingView` with `View` earlier in the code.
```diff
-    </KeyboardAvoidingView>
+    </View>
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->